src/ethereum_test_tools/common/helpers.py

```diff
@@ -104,3 +104,26 @@ def to_hash(input: int | str) -> str:
     Converts an int or str into proper 32-byte hash hex string.
     """
     return "0x" + to_hash_bytes(input).hex()
+
+
+def add_kzg_version(b_hashes, kzg_version):
+    """
+    Adds the Kzg Version to each blob hash.
+    """
+    kzg_version_hex = format(kzg_version, "02x")
```

Comment:
We could do `bytes([kzg_version])` to avoid working with strings.

Reply:
I was wondering if we could combine it with the function you added to for the precompile tests? What do you think? So we have a general function that can be used in all the 4884 tests?

With `pytest` we could even have a `common.py`/`helpers.py` (`spec.py`)  within the `eips/eip4884/` folder for all the globally used variables and helper functions related to 4844?